functional_tests/code-save/code-save-disabled.yea

```diff
@@ -0,0 +1,18 @@
+id: 0.code-save.1-disabled
```

Comment:
I'm used to seeing `.yea` files have the same exact name as their `.py` files they're testing. When the names differ, would it be useful to have a comment in the `.yea` file noting the `.py` file it's testing as well as usage? i.e for `code-save-disabled.yea` here, it would be `WANDB_DISABLE_CODE=true python code-save.py`

Reply:
Comment not needed, it is in the descriptor..
usage is:
```
$ yea run functional_tests/code-save/code-save-enabled.yea 
```
Notice in the yea file:
```
command:
  program: code-save.py
```

yea automatically finds all .yea files and all .py files that have yea annotations in the first docstring.

Reply:
Ah, gotcha. Great!